gerritcheck/check.py

```diff
@@ -133,6 +133,12 @@ def cppcheck_on_files(files, commit):
             review["labels"] = {"Code-Review": -1}
             return json.dumps(review)
 
+    # Check the return code only just now as cppcheck might still have returned
+    # some valid comments.
+    if rc != 0:
```

Comment:
So after a long pause, I was giving this some thought again. To me using a verbose mode does not really make sense in this case. I believe we all agree that in case of cppcheck / cpplint not being run correctly (exit code != 0) we must not claim "No issues found. OK", because we simply don't know. So we have to say something else, like the proposed "Did not complete successfully". But if we do that, the next immediate question is "Why?", and we should do our best to answer that question, even in non-verbose mode, by appending `out` to it. @grundprinzip, would you agree with this reasoning?